dataAccessObjects.py

```diff
@@ -161,6 +161,11 @@ def updateEditSchedule(self,targetId,displayTime,modeSn,scheSn):
             +" WHERE sche_sn={scheSn}".format(scheSn=scheSn)
         self.db.cmd(sql)
 
+    def getToCleanSchedule(self,limitCount):
```

Comment:
`getUsedSchedules` is better than `getToCleanSchedule`, because the first one is more general.